app/src/main/res/values/strings.xml

```diff
@@ -141,6 +141,23 @@
     <string name="end_time_is_less_than_start_time">End time is less than start time</string>
     <string name="you_have_to_choose_the_from_time_first">  You have to choose the from time first</string>
 
+    <!-- Validate Request Leave -->
+    <string name="the_working_time_dose_not_fit_to_the_request">The working time dose not fit to the request</string>
+    <string name="form_overdue">Form overdue</string>
+    <string name="this_is_form_request_late_time_in_is_incorrect">This is form request late, time in is incorrect!</string>
+    <string name="your_amount_tim_can_not_greater_than_max_allow_time">Your amount time can not greater than max allow time </string>
+    <string name="check_in_time_must_be_in_morning_shift">Check in time must be in morning shift</string>
+    <string name="check_in_time_must_be_in_afternoon_shift">Check in time must be in afternoon shift</string>
+    <string name="check_out_time_must_be_in_morning_shift">Check out time must be in morning shift</string>
+    <string name="check_out_time_must_be_in_afternoon_shift">Check out time must be in afternoon shift</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.check_out_time_must_be_in_afternoon_shift` appears to be unused
 Unused resources make applications larger and slow down builds.